src/browser-frontend/views/browser/chrome/tabbar/tab/tab-close-button.jsx

```diff
@@ -31,7 +31,8 @@ export default class TabCloseButton extends PureComponent {
   handleClick = (e) => {
     this.props.dispatch(PagesModelActions.tabbar.closeTabAnimated({
       pageId: this.props.pageId,
-      removePageAfterMs: 300,
+      removePageAfterMs: 200,
```

Comment:
Yes, the other transition must be a typo. I tried to get them all down to 200ms. and yes, transitionend would be better